src/IBM.WatsonDeveloperCloud/Http/Request.cs

```diff
@@ -109,6 +109,7 @@ public TaskAwaiter<IResponse> GetAwaiter()
         public async Task<T> As<T>()
         {
             HttpResponseMessage message = await this.AsMessage().ConfigureAwait(false);
+            var result = message.Content.ReadAsStringAsync().Result;

```

Comment:
Let's merge what we have here and open this up as a separate issue since it doesnt have anything to do with Visual Recognition!